README.md

```diff
@@ -30,6 +30,10 @@ management, amongst others. This option is provided as a
 just like the <a href="https://github.com/telefonicaid/fiware-cygnus/tree/master/flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks" target="_blank">other data stores</a> already supported by Cygnus.
 This option is the formal one.
 
+Since both mechanisms (the formal one using Cygnus and the minimalist one using the STH component directly) update the same database,
+it is the responsibility of the people or software in charge of creating the needed subscriptions to avoid updating the
+time series database twice. This would happen if both mechanisms are enabled for the same attribute of the same entity.
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/1f4f26ac07b0207e8361d6bb4d8f57ec536e892d Thanks @frbattid !